SCHNDR/SCHNDR-IOC-01App/Db/RIKENFE_Pumpset_Summary.template

```diff
@@ -0,0 +1,26 @@
+record (calc, "$(P)$(PV):GH") {
```

Comment:
Could you add a comment describing what this actually *represents*? It's not clear to me reading this what this record represents - except that it's some kind of summary of gatehead statuses...